internal/replication/env_test.go

```diff
@@ -265,7 +266,7 @@ func newNodeEnv(n uint32, testDir string, lg *logger.SugarLogger, clusterConfig
 	}
 
 	qBarrier := queue.NewOneQueueBarrier(lg)
-
+	pendingTxs := queue.NewPendingTxs(lg) //TODO test release blocks when not leader or error from Raft on proposal
```

Comment:
I assume this TODO and other TODO in the block creator for the upcoming PR where it will call `ReleaseWithError()`.